cypress/e2e/application-initiated-status.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Upload Candidate Resume for Application Initiated Status
+    Background:
+        Given Admin login to the system
+        Given Admin added new employee
+        Given Admin added a new Job title
+        Given Admin added a new vacancy
+        Given Admin added a new candidate
+        Given Admin ShortListed the added candidate
+        Given User log out
+
```

Comment:
The other lines follows the first line with Given should be And 
And you can use before Each hook to prepare data instead of back ground (additional note)